md/message-states.md

```diff
@@ -1,6 +1,27 @@
-# Message states
+# SMS message states
 
-All messages in the system are in one of the following states.
+## Interaction between medic-webapp and medic-gateway
+[Medic-webapp](https://github.com/medic/medic-webapp) uses [medic-gateway](https://github.com/medic/medic-gateway)
+to send SMS messages.
+
+When a report comes in, [medic-sentinel](https://github.com/medic/medic-sentinel) adds the appropriate list of
```

Comment:
Language could be clearer here.  We were previously talking about SMS, but now have switched to reports without any context or explanation for the shift.